packages/soy-gateway/src/config/infura.test.js

```diff
@@ -1,24 +1,30 @@
 const config = require('./index');
 
 describe('infura config', () => {
+  let env;
   /**
    * Sets the test environment variables
    *
    * @param {string} network - Network to set
    */
   function setEnv(network) {
     process.env.INFURA_API_KEY = 'test-value';
+
     if (network) {
       process.env.INFURA_NETWORK = network;
+    } else {
+      delete process.env.INFURA_NETWORK;
     }
   }
 
   beforeAll(() => {
+    env = JSON.parse(JSON.stringify(process.env));
```

Comment:
Really? Does the marshaling of obj -> string -> obj help in some way?

Reply:
This is, in js, the most time efficient way to clone an object without circular references.

Sense I'm playing around with deletes and stuff, I wanted to make sure the env was set back to normal and make sure that weird object references weren't the reason for a strange bug. I'm not 100% sure it's necessary.